launcher/test-framework/org.jboss.tools.common.launcher.reddeer/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,23 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Launcher Reddeer
+Bundle-SymbolicName: org.jboss.tools.common.launcher.reddeer
+Bundle-Version: 3.10.2.qualifier
+Require-Bundle: org.eclipse.ui,
+ org.jboss.tools.common.ui,
+ org.jboss.tools.common.core,
+ org.eclipse.core.runtime,
+ org.eclipse.core.databinding,
+ org.eclipse.jface.databinding,
+ org.eclipse.core.databinding.property,
+ org.eclipse.core.databinding.beans,
+ org.eclipse.core.resources,
+ org.jboss.tools.common.launcher.core,
+ org.apache.commons.io;bundle-version="2.2.0",
+ org.jboss.tools.common,
+ org.eclipse.m2e.core,
+ org.eclipse.reddeer.go;bundle-version="2.3.0",
+ org.jboss.tools.central.reddeer
+Bundle-ActivationPolicy: lazy
```

Comment:
Do you need all of these dependencies? E.g. org.eclipse.core.databinding.* are required, but I do not see them used. Are these bundles required due to central page?

Reply:
I have spent some time on this issue to figure out, what dependencies are needed. On localhost It looks like there are no dependencies needed, on Jenkins it fails with all dependencies removed. After some time I have found that there are 2 dependencies required, the rest (13 dependencies) has been removed.